keyboards/keyquest/enclave/rules.mk

```diff
@@ -0,0 +1,19 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+# Change yes to no to disable
+
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable WS2812 RGB underlight.
```

Comment:
```suggestion
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
```

Reply:
fixed on last commit. such a small thing idk how i missed it, sorry for the mistake this time it should all be fine